settings.cpp

```diff
@@ -190,6 +210,13 @@ void PTZSettings::currentChanged(const QModelIndex &current, const QModelIndex &
 		ui->ipAddressComboBox->setEnabled(true);
 		ui->udpPortSpinBox->setEnabled(true);
 	}
+	if (type == "pelco-p")
+	{
+		ui->pelcoPComboBox->setCurrentText(obs_data_get_string(cfg, "port"));
+		ui->pelcoPAddressSpinner->setValue(obs_data_get_int(cfg, "address"));
+		ui->pelcoPComboBox->setEnabled(true);
+		ui->pelcoPAddressSpinner->setEnabled(true);
```

Comment:
Here you can do:
```
ui->label_6->setText("PELCO-P Serial")
```
And then add similar text to the VISCA section. You'll probably also want to change 'label_6' to something meaningful in Qt Designer